builtin/game/chatcommands.lua

```diff
@@ -576,14 +576,27 @@ local function handle_give_command(cmd, giver, receiver, stackstring)
 	-- The actual item stack string may be different from what the "giver"
 	-- entered (e.g. big numbers are always interpreted as 2^16-1).
 	stackstring = itemstack:to_string()
+	-- easter egg when giving "forbidden" items
+	local you_hacker = itemstack:get_name() == "air" or itemstack:get_name() == "ignore"
```

Comment:
Perhaps we should actually prevent the giving of 'ignore'? After all it is not placable by a player.